cmake/compiler/msvc.cmake

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+#
+
+
+if(NOT CRT_OPT)
+ set(CRT_OPT -MD)
+endif()
+
+set(VCINCLUDES ${VCBASE}/VC/Tools/MSVC/${VCVER}/include)
+set(WINSDKINCLUDES ${WINSDKBASE}/10/Include/${UCRTVER}/ucrt ${WINSDKBASE}/8.0/Include/shared ${WINSDKBASE}/8.0/Include/um)
+
+list(APPEND ML_SYSTEM_INCLUDE_DIRECTORIES ${CMAKE_SOURCE_DIR}/3rd_party/include ${ROOT}/usr/local/include ${VCINCLUDES} ${WINSDKINCLUDES})
+
+list(APPEND ML_COMPILE_DEFINITIONS
+	_CRT_SECURE_NO_WARNINGS
+	_CRT_NONSTDC_NO_DEPRECATE
+	WIN32_LEAN_AND_MEAN
+    NOMINMAX
+	NTDDI_VERSION=0x06010000
+	_WIN32_WINNT=0x0601 Windows)
```

Comment:
This is indented with a mixture of tabs and spaces.

Is there any hard requirement to use tabs in CMake files?  If not then I think it would be best to replace tabs with spaces in all the CMake build files.